lib/Db/PollMapper.php

```diff
@@ -290,6 +294,30 @@ protected function joinOptionsForMaxDate(IQueryBuilder &$qb, string $fromAlias):
 	}
 
 
+	/**
+	 * Subquery for votes count
+	 */
+	protected function joinVotesCount(IQueryBuilder &$qb, string $fromAlias, IParameter $currentUserId, ?IParameter $answerFilter = null): IQueryBuilder {
```

Comment:
```suggestion
	protected function joinVotesCount(IQueryBuilder $qb, string $fromAlias, IParameter $currentUserId, ?IParameter $answerFilter = null): IQueryBuilder {
```
You only need a reference if you intend to replace the object with another one (`$qb = xxx`).

Reply:
Yes, right. But I would say for consistency the return should be removed, since all joins here base on references.

Reply:
But even the joins do not require the parameter to be a reference.